File: VB/CustomWizardExample/Form1.vb
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Linq
Imports System.Text
Imports System.Threading.Tasks
Imports System.Windows.Forms
Imports DevExpress.XtraReports.UserDesigner

Namespace CustomWizardExample
	Partial Public Class Form1
		Inherits XRDesignRibbonForm

		Public Sub New()
			InitializeComponent()
			DesignMdiController.AddCommandHandler(New CustomWizardCommandHandler(DesignMdiController))
			DesignMdiController.CreateNewReport()
		End Sub
	End Class
End Namespace

File: CS/CustomWizardExample/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;
using DevExpress.XtraReports.UserDesigner;

namespace CustomWizardExample {
    public partial class Form1 : XRDesignRibbonForm {
        public Form1() {
            InitializeComponent();
            DesignMdiController.AddCommandHandler(new CustomWizardCommandHandler(DesignMdiController));
            DesignMdiController.CreateNewReport();
        }
    }
}
